src/newLaunch/seed/stages/stage4.html

```diff
@@ -120,6 +120,22 @@
               </div>
             </div>
           </div>
+          <div class="infoInput csvList">
+              <div class="csvUploadInputContainer">
+              <label for="allowList"><span class="csvLabelContent">Allowlist (Optional)</span><question-mark text="Upload your allowlist in CSV format"></question-mark></label>
+              <span class="fakeLabel">
+                <input class="footer-input"
+                id="allowList"
+                type="file"
+                accept=".csv"
+                files.bind="csv"
+                placeholder="Upload your .csv allowlist..." />
+              </div>
+            </span>
+              <div >
+                <button class="upload-csv" click.delegate="parseCsv()">Upload</button>
+                </div>
+          </div>
```

Comment:
formatting off for this whole section, can you adjust please?

Reply:
how about we move and do as giald suggests: 
_At this point probably best to not having any formatter, IMO… or once dev is merged into 756, we can create a PR that format’s all files to specific config and continue with unified EditorConfig from there?_
I don't know what settings to use to format it to your expectations